ui/test/nuxeo-user-avatar.html

```diff
@@ -0,0 +1,72 @@
+<!--
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<html>
+  <head>
+    <title>nuxeo-user-avatar</title>
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes" />
+    <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+    <script src="../../../wct-browser-legacy/browser.js"></script>
+    <script src="../../../@nuxeo/nuxeo-elements/dedup.js"></script>
+    <script src="../../../@nuxeo/nuxeo-elements/test/test-helpers.js"></script>
+  </head>
+  <body>
+    <test-fixture id="nuxeo-user-avatar-fixture">
+      <template>
+        <nuxeo-user-avatar user />
```

Comment:
We shouldn't need to have that `user` there. Any special reason?

Reply:
No, no need to be here as well. Going to delete it.